### PR TITLE
Merge dev to master

### PR DESCRIPTION
**Pull request by uday-kybing**
Pulled master branch to develop and resolved conflicts

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,36 @@
+<<<<<<< HEAD
```
Comment (anusha-kybing) [inline src/App.js]: Please resolve conflicts

File: src/index.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (anusha-kybing) [inline src/index.js]: Here also resolve conflicts

File: src/About.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from  'react';
```
Comment (anusha-kybing) [inline src/About.js]: I recommend you to delete this file

**Review by anusha-kybing — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (uday-kybing) [reply src/App.js]: VSCode auto save issue.

File: src/index.js (hunk 2 above)
Comment (uday-kybing) [reply src/index.js]: same here

File: src/About.js (hunk 3 above)
Comment (uday-kybing) [reply src/About.js]: Done

**Comment by uday-kybing** [comment]
This PR is to test resolving conflicts against master branch.

**Pull request merged**